README.md

```diff
@@ -1,8 +1,31 @@
 # bca-gpa-2023
+
 An update version of [the BCA GPA browser extension](https://github.com/alebod21/bcagpa) that fixes bugs due to PowerSchool updates, and also adds a port to Firefox.
 
 Chrome Web Store Page: https://chrome.google.com/webstore/detail/bca-gpa/ofhfmclagjfikdfhjnogmcokilbikmpf
 
 Firefox Addon Page: https://addons.mozilla.org/en-US/firefox/addon/bca-gpa/
 
 ![Screenshot of extension](./screenshot.png)
+
+# Using BCA GPA on Safari
+
+BCA GPA is not available as an extension for Safari yet, but here is a method of achieving the same functionality as on Chrome or Firefox:
+
+-   Install the [Userscripts](https://apps.apple.com/ua/app/userscripts/id1463298887) extension for Safari through the Mac App Store
+    -   we will use this to run BCA GPA on Powerschool
+    -   this extension works exactly like Greasyfork or Tampermonkey and lets you run your own scripts on websites.
```

Comment:
Capitalize these

On some lines you have periods on others you don't, add to all of them if possible

Reply:
Done